.ci/scripts/run_sklearn_tests.py

```diff
@@ -37,22 +37,38 @@
     )
     args = parser.parse_args()
 
-    os.chdir(os.path.dirname(sklearn.__file__))
+    cwd = os.getcwd()
+    root_dir = os.path.dirname(sklearn.__file__)
+    os.chdir(root_dir)
 
     if os.environ["SELECTED_TESTS"] == "all":
         os.environ["SELECTED_TESTS"] = ""
 
     pytest_args = (
         "--verbose --durations=100 --durations-min=0.01 "
-        f"--rootdir={os.path.dirname(sklearn.__file__)} "
+        f"--rootdir={root_dir} "
         f'{os.environ["DESELECTED_TESTS"]} {os.environ["SELECTED_TESTS"]}'.split(" ")
     )
+
+    if os.getenv("COVERAGE_RCFILE"):
+        pytest_args += (
+            f'--cov=onedal --cov=sklearnex --cov-config={os.environ["COVERAGE_RCFILE"]} '
+            "--cov-report=term".split(" ")
+        )

```

Comment:
```suggestion
    if rc := os.getenv("COVERAGE_RCFILE"):
        pytest_args += (
            f'--cov=onedal --cov=sklearnex --cov-config={rc} '
            "--cov-report=term".split(" ")
        )

```

Since we no longer support python < 3.8 we can start using the walrus operator.